packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -132,6 +153,24 @@ export class Accordion {
   //
   //--------------------------------------------------------------------------
 
-  private sortItems = (items: any[]): any[] =>
-    items.sort((a, b) => a.position - b.position).map((a) => a.item);
+  private updateAccordionItems = (addedAccordionItem?: HTMLCalciteAccordionItemElement): void => {
+    if (addedAccordionItem) {
+      this.accordionItems.push(addedAccordionItem);
+    } else {
+      const accordionItemsQueryArray = Array.from(
+        this.el.querySelectorAll("calcite-accordion-item")
+      );
+      this.accordionItems = accordionItemsQueryArray;
```

Comment:
What is the purpose of pushing `addedAccordionItems` into `accordionItems` if `accordionItems` is being re-referenced to `accordionItemsQueryArray` directly afterwards?

Reply:
Looks like it isn't needed anymore. Removing position removed the need for accordionItems being an array of objects, which makes that bit redundant.